unit_tests/test_redis_aggregator.py

```diff
@@ -0,0 +1,329 @@
+# pylint: disable=missing-function-docstring, protected-access, no-member
+import json
+import uuid
+from unittest.mock import patch, PropertyMock, MagicMock, call
+import pytest
+from gsy_e_sdk.constants import LOCAL_REDIS_URL
+
+from gsy_e_sdk.redis_aggregator import RedisAggregator, RedisAggregatorAPIException
+
+TEST_AGGREGATOR_NAME = "TestAgg"
+
+TEST_TRANSACTION_ID = str(uuid.uuid4())
+TEST_AGGREGATOR_UUID = str(uuid.uuid4())
+
+TEST_DEVICE_UUID_1 = str(uuid.uuid4())
+TEST_DEVICE_UUID_2 = str(uuid.uuid4())
+
+TEST_BATCH_COMMAND_DICT = {
+    TEST_DEVICE_UUID_1: ["TEST_BATCH_COMMAND"],
+    TEST_DEVICE_UUID_2: ["TEST_BATCH_COMMAND"]
+}
+
+TEST_TRANSACTION_ID_RESPONSE = "TEST_TRANSACTION_ID_RESPONSE"
+TEST_TRANSACTION_ID_BUFFER = {
+    TEST_TRANSACTION_ID: {"transaction_id": TEST_TRANSACTION_ID,
+                          "response": TEST_TRANSACTION_ID_RESPONSE}
+}
+
+TEST_RESPONSE = {"status": "ready",
+                 "command": "offer",
+                 "offer": {"energy": 1,
+                           "price": 1,
+                           "seller": "test_seller"},
+                 "bid": {"energy": 1,
+                         "price": 1,
+                         "buyer": "test_buyer"},
+                 }
+
+TEST_TRANSACTION_ID_RESPONSE_BUFFER = {TEST_TRANSACTION_ID: TEST_RESPONSE}
+
+TEST_BATCH_RESPONSES = {"TEST_ASSET_UUID": [TEST_RESPONSE]}
```

Comment:
We can get rid of this one too, the reason I push for decreasing the number of these variables because we would probably end up debugging where all of them are used/defined and this may lead to some issues

It would be great if you can remove all variables that are only used once, and remove these global variables that are only used within 1 test case